bin/make-it-executable.js

```diff
@@ -3,10 +3,20 @@ const path = require('path');
 const { promisify } = require('util');
 
 const ROOT = path.join(__dirname, '../');
-const scriptPaths = require(path.join(ROOT, './package.json')).bin;
+let scriptPaths = require(path.join(ROOT, './package.json')).bin;
 
 const INSERT = '#!/usr/bin/env node';
 
+/**
+ * The forEach loop below requires an object with script paths as properties to loop over. When the `bin` property in
+ * the package.json only has a single property, then the object is converted to a string upon a `yarn install` call
+ * (see: https://docs.npmjs.com/cli/v9/configuring-npm/package-json#bin). The string to object transformation here thus
+ * allows us to support 1+ properties whilst keeping the loop below.
+ */
+if (typeof scriptPaths === 'string') {
+  scriptPaths = { "datadog-ci": scriptPaths }
+}
+
```

Comment:
This change was "required" because of the adjustment made to the `bin` property in the `package.json`. I'm curious to know if a different approach would be preferred (e.g.: ignoring the change made to the `package.json` after a `yarn install`).

This approach at least allows us to expand the `scriptPaths` defined in the `bin` property later, from 1 to 2+ paths, as the loop to handle multiple items will still be there.